go/worker/compute/executor/committee/node.go

```diff
@@ -1167,9 +1167,14 @@ func (n *Node) proposeBatchLocked(processedBatch *processedBatch) {
 	}()
 	storageCommitLatency.With(n.getMetricLabels()).Observe(time.Since(start).Seconds())
 
-	if err != nil {
-		n.abortBatchLocked(err)
-		return
+	if storageErr != nil {
+		n.logger.Error("storage failure, sibmitting failure indicating commitment",
+			"err", storageErr,
+		)
+		proposedResults.RakSig = nil
+		proposedResults.Header.IORoot = nil
+		proposedResults.Header.StateRoot = nil
+		proposedResults.Failure = commitment.FailureStorageUnavailable
```

Comment:
Could also make a method like `proposedResults.SetFailure(commitment.FailureStorageUnavailable)` which would take care of nilifying the required fields?

Reply:
good idea, done!